index.html

```diff
@@ -7,58 +7,268 @@
     <link rel="stylesheet" href="Styles.css" />
```

Comment:
- [ ] Please make sure that the background of the toolbar section matches the design currently there is a difference between the background in the design and in your code.
Figma
![image](https://user-images.githubusercontent.com/74744787/227737230-f3cef7fa-9dfc-40c7-a7b6-98400ac4c10c.png)
Your design 
![image](https://user-images.githubusercontent.com/74744787/227737246-2c9b91dd-45b0-4073-8a70-46e8f2c009d6.png)